### PR TITLE
Increase test utils

### PR DESCRIPTION
**Pull request by keiya01**
Test coverage is almost 100% in this PR 🎉

### CODE REVIEW

**Comment by bluebill1049** [comment]
wow! 💯

File: src/utils/onDomRemove.test.ts [hunk 1]
```diff
@@ -13,4 +20,59 @@ describe('onDomRemove', () => {
       subtree: true,
     });
   });
+
+  it('should call the mutation callback', () => {
```
Comment (bluebill1049) [inline src/utils/onDomRemove.test.ts:24]: holy carp! you did this hahah, i gave up trying to get this one covered with tests.

**Comment by bluebill1049** [comment]
wow https://coveralls.io/github/bluebill1049/react-hook-form?branch=master 98%!

**Comment by keiya01** [comment]
Finally🎉

**Pull request merged**